purescript.cabal

```diff
@@ -39,6 +39,7 @@ extra-source-files:
   INSTALL.md
   CONTRIBUTORS.md
   CONTRIBUTING.md
+  .hspec
```

Comment:
@hdgarrood I'm not sure if it is a good idea to include the `.hspec` file in the generated dist tarball.  By doing so, tests will fail for users that run the tests for a release if they don't have the latest (or as of now even unreleased) version of `hspec`.

Reply:
Ah of course, thanks for pointing that out. I'm inclined not to worry though, because I don't think it's particularly important in our case for the tests to work in any other context than running `stack test` inside the repo. But I guess if this causes problems we can always take it out again and supply the flag on the command line in the CI script.

Reply:
Yes, I agree.  I don't think it's particularly important neither.  For my packages I used to not care much, but I had people complain about failing tests (e.g. `stackage` runs tests by default). I guess as long as nobody complains, it's fine.

> we can always take it out again and supply the flag on the command line in the CI script.

To be clear, I don't see any issue with the `.hspec` file in the repo, just adding it to the `.cabal` package could cause issues without providing much benefit.

Reply:
Right - to clarify, the reason I was tempted to add .hspec here in the first place was that our CI runs tests in a source distribution to ensure that we don’t end up publishing a tarball which is missing files that are necessary for the package to build or something, so just adding .hspec to the repo isn’t enough to get the times shown in CI.

Reply:
Ok, that makes sense.